setup.py

```diff
@@ -132,6 +132,7 @@ def run(self):
     ],
     "development": [
         "Flask-DebugToolbar==0.9.0",
+        "watchdog==0.8.3"
```

Comment:
Please add comma (`,`) at the end of the line so next time somebody adds a package we get again nice one line diff.